packages/components/tooltip/src/tooltip.tsx

```diff
@@ -283,15 +302,17 @@ const Tooltip = (props: TTooltipProps) => {
         // for seo and accessibility, we add the tooltip's title
         // as a native title when the title is hidden
         title:
-          !tooltipIsOpen && typeof props.title === 'string'
+          !tooltipIsOpen &&
+          state !== 'entering' &&
```

Comment:
Adding the new `entering` state here helps with correctly handling children's `title`.